classes/utils/Biome.py

```diff
@@ -0,0 +1,95 @@
+from enum import Enum
+
+
+class Biome(Enum):
+
+    def __init__(self, id, string):
+        self.id = id
+        self.string = string
+
+    def id_to_biome(id):
+        for biome in Biome:
+            if biome.id == id:
+                return biome
+        return None
+
+    def name_to_biome(name):
+        for biome in Biome:
+            if biome.name == name:
+                return biome
+        return None
+
+    def string_to_biome(string):
+        for biome in Biome:
+            if biome.biome == string:
+                return biome
+        return None
+
+    INVALID_BIOME = (-1, 'Invalid Biome')
+    OCEAN = (0, 'Ocean')
+    PLAINS = (1, 'Plain')
+    DESERT = (2, 'Desert')
+    MONTAINS = (3, 'Montains')
+    EXTREME_HILLS = (3, 'ExtremeHills')
+    FOREST = (4, 'Forest')
+    TAIGA = (5, 'Taiga')
+    SWAMPLAND = (6, 'Swampland')
+    RIVER = (7, 'River')
+    HELL = (8, 'Hell')
+    NETHER = (8, 'Nether')
+    SKY = (9, 'Sky')
+    END = (9, 'End')
+    FROZEN_OCEAN = (10, 'FrozenOcean')
+    FROZEN_RIVER = (11, 'FrozenRiver')
+    ICE_PLAINS = (12, 'IcePlains')
+    TUNDRA = (12, 'Tundra')
+    ICE_MOUNTAINS = (13, 'IceMountains')
+    MUSHROOM_ISLAND = (14, 'MushroomIsland')
+    MUSHROOM_SHORE = (15, 'MushroomShore')
+    BEACH = (16, 'Beach')
+    DESERT_HILLS = (17, 'DesertHills')
+    FOREST_HILLS = (18, 'ForestHills')
+    TAIGA_HILLS = (19, 'TaigaHills')
+    EXTREME_HILLS_EDGE = (20, 'ExtremeHillsEdge')
+    JUNGLE = (21, 'Jungle')
+    JUNGLE_HILLS = (22, 'JungleHills')
+    JUNGLE_EDGE = (23, 'JungleEdge')
+    DEEP_OCEAN = (24, 'DeepOcean')
+    STONE_BEACH = (25, 'StoneBeach')
+    COLD_BEACH = (26, 'ColdBeach')
+    BIRCH_FOREST = (27, 'BirchForest')
+    BIRCH_FOREST_HILLS = (28, 'BirchForestHills')
+    ROOFED_FOREST = (29, 'RoofedForest')
+    COLD_TAIGA = (30, 'ColdTaiga')
+    COLD_TAIGA_HILLS = (31, 'ColdTaigaHills')
+    MEGA_TAIGA = (32, 'MegaTaiga')
+    MEGA_TAIGA_HILLS = (33, 'MegaTaigaHills')
+    EXTREME_HILLS_PLUS = (34, 'ExtremeHillsPlus')
+    SAVANNA = (35, 'Savanna')
+    SAVANNA_PLATEAU = (36, 'SavannaPlateau')
+    MESA = (37, 'Mesa')
+    MESA_PLATEAU_F = (38, 'MesaPlateauF')
+    MESA_PLATEAU = (39, 'MesaPlateau')
+
```

Comment:
Nether and other biomes (such as Void) should also be added before we forget that.